colors/dracula.vim

```diff
@@ -207,15 +207,16 @@ call s:h('StatusLineTermNC', s:none, s:bglight)
 call s:h('WildMenu', s:bg, s:purple, [s:attrs.bold])
 call s:h('CursorLine', s:none, s:subtle)
 
+call s:h('DiffAdd', s:green)
+call s:h('DiffChange', s:orange, s:none)
+call s:h('DiffText', s:bg, s:orange)
+call s:h('DiffDelete', s:red, s:bgdark)
+
 hi! link ColorColumn  DraculaBgDark
 hi! link CursorColumn CursorLine
 hi! link CursorLineNr DraculaYellow
-hi! link DiffAdd      DraculaGreen
 hi! link DiffAdded    DiffAdd
-hi! link DiffChange   DraculaDiffChange
-hi! link DiffDelete   DraculaDiffDelete
 hi! link DiffRemoved  DiffDelete
```

Comment:
What about `DiffAdded` and `DiffRemoved`? Do those not need to be highlighted this way?

Reply:
I don't really know the internals, so all I can say, that in practice they do not :) For the sake of symmetry they could be, but I don't really know if there are any downsides you'd need to consider.

Reply:
Hmm yeah, can you just make those the same as the other diff ones just for future-proofing?

Reply:
should be done